code/modules/mob/mob_defines.dm

```diff
@@ -54,6 +54,8 @@
 	var/eye_damage = 0//Living, potentially Carbon
 	var/lastpuke = 0
 
+	var/circulatory = 0 // Some simple mobs will receive damage from disabler bolts
```

Comment:
the tl;dr is adding a var to mobs for this is awful when all that needs changing is a proc and a varlist